userdocs/src/usage/gitops.md

```diff
@@ -314,15 +371,54 @@ monitoring             prometheus-operator-prometheus-node-exporter-tllwl
 monitoring             prometheus-prometheus-operator-prometheus-0               3/3     Running                      1          72s
 ```
 
+If you fetch the latest changes to your configuration repository,
+you will see that eksctl has updated it with the templated files from the
+Quick Start. The next time Flux syncs from this repo, it will start updating
+the cluster with the current configuration.
+
+You will now also have the following components running in your cluster:
```

Comment:
Forgot to include this suggestion but I prefer a link to `eks-quickstart-app-dev/README.md` rather than copy pasting

Reply:
yep that would be better 👍